### DISCUSSION TITLE
3 error massages of yamacs

### DISCUSSION THREAD

**Discussion by wonwill** [YAMACS-SML/YAMACS#6] [Q&A] [answered]
Thanks to your YAMACS, I have been enjoying the gromacs md simulation.
But I got three kinds of error messages from yamacs.

The 1 and 2 messages appeared at the very end of the mdrun.
The error 3 message (third image) happened and the plugin also stopped unexpectedly.

If I want to "continue" the previous gmx mdrun, how can I do it?
Should I try to do mdrun from the beginning?
For example, as shown in unfinished log below, I want to add few mdrun simulations.

Thanks a lot in advance.

error 1
![yamacs-error1](https://github.com/YAMACS-SML/YAMACS/assets/5327542/7c304076-26aa-446d-b887-52113635cb59)
error 2
![yamacs_error2](https://github.com/YAMACS-SML/YAMACS/assets/5327542/03e54afb-05fd-4039-9ba0-a2cc0bc48236)
error 3
<img width="798" alt="yamacs_error3" src="https://github.com/YAMACS-SML/YAMACS/assets/5327542/5bd968f8-2553-430a-a503-ec4a93a1731c">

unfinished log
<img width="537" alt="yamacs_mdrun_output" src="https://github.com/YAMACS-SML/YAMACS/assets/5327542/8ddb0ac6-52ab-401e-a727-2b36872b68a5">

**Comment by YAMACS-SML** [comment, marked as answer]
The first error indicates that YASARA did not find the "tempo.ini" file inside the plg folder. I suggest you to delete all the ".ini" file from the plg folder and restart the calculation from the beginning.

The second error indicates that the shutil module is not installed in your python environment. I suggest you to install the shutil module using "pip install pytest-shutil" command in your python environment. 

The third error indicates that YASARA stopped responding in your system. May be you need to update your YASARA structure version. This error may happen due to several reason. May be you executing any parallel job along with YAMACS, and that is causing the error. But try to update and restart YASARA.

**Comment by wonwill** [comment]
Thanks for your quick, kind, and detailed reply. It is very helpful for me.
